osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -1363,6 +1363,34 @@
         "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/inj.svg"
       },
       "coingecko_id": "injective-protocol"
-    }
+    },
+    {
+      "description": "The native token of Marble DEX on Juno Chain",
+      "type_asset": "cw20",
+      "address": "juno1y9rf7ql6ffwkv02hsgd4yruz23pn4w97p75e2slsnkm0mnamhzysvqnxaq",
+      "denom_units": [
+        {
+          "denom": "ibc/DB9755CB6FE55192948AE074D18FA815E1429D3D374D5BDA8D89623C6CF235C3",
+          "exponent": 0
+        },
+        {
+          "denom": "block",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/DB9755CB6FE55192948AE074D18FA815E1429D3D374D5BDA8D89623C6CF235C3",
+      "name": "Block",
+      "display": "block",
+      "symbol": "BLOCK",
+      "ibc": {
+        "source_channel": "channel-47",
+        "dst_channel": "channel-169",
+        "source_denom": "block"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/block.png",
```

Comment:
Please add png to PR

Reply:
Alternatively (preferred), please specify the following for the png URI:
https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/block.png